resources/zib/zib-GlasgowComaScale.xml

```diff
@@ -0,0 +1,256 @@
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://loinc.org"/>
+          <code value="35088-4"/>
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.effective[x]">
+      <path value="Observation.effective[x]"/>
+      <slicing>
+        <discriminator>
+          <type value="type"/>
+          <path value="$this"/>
+        </discriminator>
+        <rules value="open"/>
+      </slicing>
+    </element>
+    <element id="Observation.effective[x]:effectiveDateTime">
+      <path value="Observation.effective[x]"/>
+      <sliceName value="effectiveDateTime"/>
+      <short value="GlasgowComaScaleDateTime"/>
+      <definition value="Time at which the EMV score was determined."/>
+      <alias value="GlasgowComaScaleDatumTijd"/>
+      <type>
+        <code value="dateTime"/>
+      </type>
+      <mapping>
+        <identity value="zib-glasgowcomascale-v3.2-2020EN"/>
+        <map value="NL-CM:12.8.8"/>
+        <comment value="GlasgowComaScaleDateTime"/>
+      </mapping>
+    </element>
+    <element id="Observation.value[x]">
+      <path value="Observation.value[x]"/>
+      <slicing>
+        <discriminator>
+          <type value="type"/>
+          <path value="$this"/>
+        </discriminator>
+        <rules value="open"/>
+      </slicing>
+    </element>
+    <element id="Observation.value[x]:valueInteger">
+      <path value="Observation.value[x]"/>
+      <sliceName value="valueInteger"/>
+      <short value="TotalScore"/>
+      <definition value="The sum of the EMV scores, expressed as a number on a scale from 3-15. The value does not have to be recorded, as it can always be calculated based on the individual scores, and as recording the total is not common for a difficult to determine individual score."/>
+      <alias value="TotaalScore"/>
+      <type>
+        <code value="integer"/>
+      </type>
+      <minValueInteger value="3"/>
+      <maxValueInteger value="15"/>
+      <mapping>
+        <identity value="zib-glasgowcomascale-v3.2-2020EN"/>
+        <map value="NL-CM:12.8.10"/>
+        <comment value="TotalScore"/>
+      </mapping>
+    </element>
+    <element id="Observation.note">
+      <path value="Observation.note"/>
+      <max value="1"/>
+    </element>
+    <element id="Observation.note.text">
+      <path value="Observation.note.text"/>
+      <short value="Comment"/>
+      <definition value="Comment on (the context of) the EMV score measurement, such as any problems or factors that may influence interpretation."/>
+      <alias value="Toelichting"/>
+      <mapping>
+        <identity value="zib-glasgowcomascale-v3.2-2020EN"/>
+        <map value="NL-CM:12.8.9"/>
+        <comment value="Comment"/>
+      </mapping>
+    </element>
+    <element id="Observation.component">
+      <path value="Observation.component"/>
+      <slicing>
+        <discriminator>
+          <type value="pattern"/>
```

Comment:
I thought pattern was more correct (because well, it is a pattern isn't it), but https://hl7.org/FHIR/profiling.html (so R5) actually deprecates pattern as it was the same as value, so I will change it